src/Sylius/Bundle/CoreBundle/Settings/TaxationSettingsSchema.php

```diff
@@ -64,6 +62,10 @@ public function buildForm(FormBuilderInterface $builder)
                 'required' => false,
                 'label' => 'sylius.form.settings.taxation.default_tax_zone',
             ])
+            ->add('default_tax_calculation_strategy', 'sylius_tax_calculation_strategy_choice', [
```

Comment:
Having a default tax calculation strategy on a per channel basis would be useful and offers a lot more power and flexibility. :+1: